### DISCUSSION TITLE
加载的 texture 如何设置宽高？

### DISCUSSION THREAD

**Discussion by RJM1996** [galacean/discussion#39] [Engine] [answered]
使用 engine.resourceManager.load 加载 texture 时如何设置宽高呢？

**Comment by cptbtptpbcptdtptp** [comment]
texture 的宽高是只读的，可以描述一下你的需求吗？

**Reply by RJM1996** [reply]
比如使用 texture 加载了一张图片，想要设置图片的大小，应该如何设置呢

**Reply by cptbtptpbcptdtptp** [reply, marked as answer]
如果这个设置是一次性的（即只希望他在特定的情景显示自定义尺寸），则可以通过 SpriteRenderer.width 和 SpriteRenderer.height 设置显示宽高，如果希望这个设置是通用的，可以通过设置 Sprite.width 和 Sprite.height 来显示。
Texture 是原始资产，一般不支持也不建议修改原始资产的信息。
